### PR TITLE
Support wildcard "*" and "?" for <source-path>

### PR DESCRIPTION
**Pull request by Fiile**
- Use "*" and "?" to match local files.
- Upload matched patterns in one request.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -22,6 +22,37 @@ System-Wide::
 On Windows systems, run it in a command-prompt window with administrator
 privileges, and leave out sudo.
 
+Configure The Command Completion
```
Comment (Xuanwo) [inline README.rst]: Split into another commit.

File: qingstor/qsctl/buckets_completion.py [hunk 2]
```diff
@@ -23,31 +26,47 @@ def complete(output, cmdline, point=None):
...
+                qs_count += 1
+        if qs_count > 1:
+            return 0
+        if cmd_args[1] in ["ls", "mb", "presign", "rb", "rm"]:
             for x in output["buckets"]:
                 if x["name"].encode("utf-8").startswith(current_arg):
                     bucket_list.append((x["name"] + "/").encode("utf-8"))
-            print(" \n".join(bucket_list))
-        else:
+            if not len(bucket_list):
```
Comment (Xuanwo) [inline qingstor/qsctl/buckets_completion.py]: We can transform these code into:

```python
if len(bucket_list):
    if cmdline.endwith(" "):
        return 0
    print(xxxxx)
```

File: qingstor/qsctl/commands/transfer.py [hunk 3]
```diff
@@ -524,7 +524,9 @@ def send_request(cls):
         transfer_flow = cls.get_transfer_flow()
 
         if transfer_flow == "LOCAL_TO_QS":
-            if (cls.command == "sync") or (cls.options.recursive is True):
+            if len(cls.options.source_path) > 1:
```
Comment (Xuanwo) [inline qingstor/qsctl/commands/transfer.py]: What will happen if I use:

```
qsctl sync aaa bbb ccc *.txt qs://xxx
```

**Comment by Fiile** [comment]
@Xuanwo

**Comment by Fiile** [comment]
@Xuanwo
